.changeset/dry-bulldogs-push.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lighting-beetle/lighter-react-scripts': minor
```

Comment:
Aaaaa toto je patch! Fix-nem to v tom #233

Reply:
🤦‍♂ neviem odpovedat na otazky

Reply:
Ja som zas nedaval pozor poriadne ;)